.github/copilot-instructions.md

```diff
@@ -103,6 +103,9 @@
 
 ## Active Technologies
 
+- TypeScript 5.3 on Node.js 20.15.1 (Volta-managed) + NestJS 10 (GraphQL + dependency injection), Apollo Server 4, TypeORM 0.3, in-house authorization services (`@domain/common/whiteboard`, `@services/whiteboard-integration`) (001-toggle-whiteboard-guest)
```

Comment:
this already exists in another form. Unify with the other one or revert the change.
In general I am not expecting the instructions to be updated per feature. We have to update some instruction to not change this file.